### PR TITLE
doc: Add shouldExist for users

### PR DESCRIPTION
**Pull request by cgwalters**
This addition to the docs was missed in the original PR
https://github.com/coreos/ignition/pull/1014

### CODE REVIEW

File: doc/configuration-v3_2_experimental.md [hunk 1]
```diff
@@ -159,6 +159,7 @@ The Ignition configuration is a JSON document conforming to the following specif
     * **_noUserGroup_** (boolean): whether or not to create a group with the same name as the user. This only has an effect if the account doesn't exist yet.
     * **_noLogInit_** (boolean): whether or not to add the user to the lastlog and faillog databases. This only has an effect if the account doesn't exist yet.
     * **_shell_** (string): the login shell of the new account.
+    * **_shouldExist_** (boolean) If set, the target user will be deleted (if present) and not created.  Defaults to false.
```
Comment (sohankunkerkar) [inline doc/configuration-v3_2_experimental.md:162]:
```suggestion
    * **_shouldExist_** (boolean) If set, the target user will be deleted (if present) and not created.  Defaults to true.
```

Comment (cgwalters) [reply doc/configuration-v3_2_experimental.md]: Hah yes thanks!

File: doc/configuration-v3_2_experimental.md [hunk 2]
```diff
@@ -159,6 +159,7 @@ The Ignition configuration is a JSON document conforming to the following specif
     * **_noUserGroup_** (boolean): whether or not to create a group with the same name as the user. This only has an effect if the account doesn't exist yet.
     * **_noLogInit_** (boolean): whether or not to add the user to the lastlog and faillog databases. This only has an effect if the account doesn't exist yet.
     * **_shell_** (string): the login shell of the new account.
+    * **_shouldExist_** (boolean) If set to false, the target user will be deleted (if present) and not created.  Defaults to true.
```
Comment (arithx) [inline doc/configuration-v3_2_experimental.md:162]:
```suggestion
    * **_shouldExist_** (boolean) whether or not the user with the specified `name` should exist. If omitted, it defaults to true. If false Ignition will delete the specified user.
```

How about this to more closely match the other `shouldExist` docs lines?

**Comment by sohankunkerkar** [comment]
> Looks like it's also missing for `groups` would you mind adding it there as well?

Nice catch! I forgot to verify that detail

**Comment by cgwalters** [comment]
> Looks like it's also missing for groups would you mind adding it there as well?
Done

**Review by jlebon — APPROVED** [review]
LGTM; restarted CI on this.

**Pull request merged**